dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESMappingAPITest.java

```diff
@@ -75,11 +97,126 @@ public static void prepare() throws Exception {
         contentTypeAPI = APILocator.getContentTypeAPI(user);
         contentletAPI = APILocator.getContentletAPI();
         fieldAPI = APILocator.getContentTypeFieldAPI();
+        folderAPI = APILocator.getFolderAPI();
         languageAPI = APILocator.getLanguageAPI();
         language = languageAPI.getDefaultLanguage();
         relationshipAPI = APILocator.getRelationshipAPI();
     }
 
+    @Test
+    public void test_toMap_fileasset_txt_shouldSuccess() throws Exception {
+
+        final ESMappingAPIImpl esMappingAPI    = new ESMappingAPIImpl();
+        final Host host = APILocator.getHostAPI().findDefaultHost(user, false);
```

Comment:
does this test fail with an empty starter? why don't you use `new SiteDataGen().nextPersisted();` to get a new host?

Reply:
It is working fine